.github/actions/travis-env-load/action.yml

```diff
@@ -1,25 +0,0 @@
-name: "travis-env-load"
```

Comment:
this action it's actually still useful, I would keep it but switch to a better name which doesn't mention travis?

`env-load-from-yaml`